apps/src/dance/Dance.js

```diff
@@ -131,6 +131,11 @@ Dance.prototype.init = function(config) {
     this.studioApp_.init(config);
     this.studioAppInitPromiseResolve();
 
+    // Manually trigger a 'resize' after the relevant React components have mounted
+    // to ensure consistent sizing/scaling and prevent overflow after instructions
+    // have rendered.
+    this.studioApp_.resizeVisualization();
```

Comment:
aside: hopefully one day we can simplify this behaviour and use React top to bottom!

Reply:
yes, hopefully soon! i have to flesh out the details, but that work is tracked as [STAR-1621](https://codedotorg.atlassian.net/browse/STAR-1621)